package.json

```diff
@@ -2,25 +2,17 @@
   "private": true,
   "repository": "https://github.com/thunderstore-io/thunderstore-ui",
```

Comment:
From the [docs](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository):
> The URL should be a publicly available (perhaps read-only) url that can be handed directly to a VCS program without any modification. It should not be a url to an html project page that you put in your browser. It's for computers.

This should probably end in `.git`